src/fidesops/ops/tasks/__init__.py

```diff
@@ -47,7 +66,15 @@ def _create_celery(config_path: str = config.execution.celery_config_path) -> Ce
 
 def start_worker() -> None:
     logger.info("Running Celery worker...")
-    celery_app.worker_main(argv=["worker", "--loglevel=info", "--concurrency=2"])
+    default_queue_name = celery_app.conf.get("task_default_queue", "celery")
+    celery_app.worker_main(
+        argv=[
+            "worker",
+            "--loglevel=info",
+            "--concurrency=2",
+            f"--queues={default_queue_name},{EMAIL_QUEUE_NAME}",
```

Comment:
ok so it didn't need to be comma separated, we just needed to use `queues` was your findings?